googleapiclient/discovery.py

```diff
@@ -201,10 +201,8 @@ def build(serviceName,
   if resp.status >= 400:
     raise HttpError(resp, content, uri=requested_url)
 
-  try:
+  if isinstance(content, bytes):
     content = content.decode('utf-8')
-  except AttributeError:
-    pass
```

Comment:
Well - theoretically another http client (or proxy) could encode response in a different character set and to pass the data properly to this code the application would have to decode the bytes into unicode and the encode them in utf-8 before passing them to this function which would again decode it.  I think it's better to allow unicode objects (str or unicode) and try to handle all bytes inputs as utf-8.